src/main/kotlin/gropius/model/common/ExtensibleNode.kt

```diff
@@ -0,0 +1,21 @@
+package gropius.model.common
+
+import com.expediagroup.graphql.generator.annotations.GraphQLDescription
+import com.expediagroup.graphql.generator.annotations.GraphQLType
+import io.github.graphglue.model.DomainNode
+import io.github.graphglue.model.Node
+
+@DomainNode
+@GraphQLDescription("Entity which provides dynamic extension fields.")
+abstract class ExtensibleNode : Node() {
+
+    @GraphQLDescription("Value of an extension field by name of the extension field.")
+    @GraphQLType("JSON")
+    fun extensionField(
+        @GraphQLDescription("Name of the extension field")
+        name: String
```

Comment:
these fields are meant to be used directly by an external program, which needs to save some additional information in our system  
The general problem is, that these fields are saved directly stringified as property. Therefore, I would recomend keeping the size of this rather small, and only allowing necessary information.
I would not recommend the JSON schema, as the external program should already know what it's doing. If it really needs the schema, it can put the schema directly in the saved object.  
For the description, I could see the usecase that this hepls the user to remove outdated extension fields, but it may still lead to a lot of unnecessary data on the nodes (please consider that these are always loaded). I would recommend that applications write the extension field sth like application.key, and then the user could simply look in the documentation to see for what the field is.  
Last, automatic processing of these fields is a non-goal, for this, we have the templatedFields on some nodes.